sources/data-node/api/v2/trading_data.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+option go_package = "code.vegaprotocol.io/protos/data-node/api/v2";
+
+import "vega/vega.proto";
+import "google/protobuf/wrappers.proto";
+
+package datanode.api.v2;
+
+service TradingDataService {
```

Comment:
Will this be the final service name or do we want to start doing smaller, categorized services?

Reply:
It's up for discussion I guess but I'm not sure it's worth the extra faff, I'm going to commit it for now :)

Reply:
I think Emil suggestion is good. let's try to follow the guidelines he's defined.